pkg/koordlet/util/system/resctrl.go

```diff
@@ -560,6 +575,21 @@ func CheckAndTryEnableResctrlCat() error {
 	return nil
 }
 
+func InitCatGroupIfNotExist(group string) error {
```

Comment:
It seems never to be used.

Reply:
This function is used in #1974 as the util function, I can remove this function in this pr.